test/apiv2/test-apiv2

```diff
@@ -220,21 +215,6 @@ function jsonify() {
     (IFS=','; echo "{${settings_out[*]}}")
 }
 
-#######
-#  t_timeout  #  Timeout wrapper for test helper
```

Comment:
Yeah, I added this. Sorry. Seemed to be a good solution at that time. Nice, it can be done even better.

Reply:
@jmguzik no apology needed! You added good tests to a test suite that is depressingly complex. Thank you. I need to figure out a way to be more proactive on review.